ch-intro/index.rst

```diff
@@ -93,4 +93,4 @@ The following changes and additions have been made from vanilla Sphinx:
 Documents Using sphinxtr
 ========================
 
-* `Jeff Terrace's PhD Thesis <http://jeffterrace.com/thesis/>`_
```

Comment:
http://jeffterrace.com/thesis/ still seems to work fine for me, what am I missing?

Reply:
Oh, I see, the followup links don't work.